lib/ecto/postgres/pg_framestamp_migrations.ex

```diff
@@ -486,11 +641,19 @@ defpgmodule Vtc.Ecto.Postgres.PgFramestamp.Migrations do
     Postgres.Utils.create_plpgsql_function(
       private_function(:sub, Migration.repo()),
       args: [a: :framestamp, b: :framestamp],
-      declares: [seconds: {:rational, "(a).seconds - (b).seconds"}],
+      declares: subtraction_declares(),
       returns: :framestamp,
       body: """
-      IF (a).rate === (b).rate THEN
-        RETURN (seconds, (a).rate)::framestamp;
+      IF #{sql_check_rate_equal(:a, :b)} THEN
+        #{Fragments.sql_inline_simplify(:numerator, :denominator, :greatest_denom)}
+
+        RETURN (
+          numerator,
+          denominator,
+          (a).__rate_n,
+          (a).__rate_d,
+          (a).__rate_tags
+        )::framestamp;
       ELSE
         #{sq_raise_mised_frame_arithmetic_error(:framestamp, :-)}
```

Comment:
should this be `sql_` and `missed_`? this is in a few places, so will leave just this one note.
```suggestion
        #{sql_raise_missed_frame_arithmetic_error(:framestamp, :-)}
```

Reply:
Whoops, was supposed to be `sq_raise_mixed_frame_arithmetic_error`.

In 3401777c12a844fe178f7d9fa585986d8819b2fc.

Reply:
out of curiosity, what is `sq` short for? i guess i'm mentally trying to force it to `sql` since `sql_check_rate_equal` is right below.

Reply:
Apparently my brain is incapable of seeing typos.

Fixed in 9a6e2fd11b6bfe394adf517616b636af3527cd3b